ddtrace/contrib/django/utils.py

```diff
@@ -282,15 +281,21 @@ def _after_request_tags(pin, span, request, response):
             _set_resolver_tags(pin, span, request)
 
             response_headers = dict(response.items()) if response else {}
+            raw_uri = url
+            if raw_uri and request.META.get("QUERY_STRING"):
+                raw_uri += "?" + request.META["QUERY_STRING"]
             trace_utils.set_http_meta(
                 span,
                 config.django,
                 method=request.method,
                 url=url,
+                raw_uri=raw_uri,
                 status_code=status,
                 query=request.META.get("QUERY_STRING", None),
                 request_headers=request_headers,
                 response_headers=response_headers,
+                request_cookies=request.COOKIES,
+                request_path_params=request.resolver_match.kwargs if request.resolver_match is not None else None,
```

Comment:
based on the description I think this is wrong. the data you want I think we set in `_set_resolver_tags`

Reply:
That one's on me - I mixed two addresse, let me update the docstring